src/converge.js

```diff
@@ -31,10 +34,10 @@ var curryN = require('./curryN');
  */
 module.exports = curryN(3, function(after) {
   var fns = _slice(arguments, 1);
-  return function() {
+  return arity(max(pluck('length', arguments)), function() {
```

Comment:
> as with `uncurryN`, should this use `curryN`?

Yes, it should. Good catch. :)

> The first function -- the join -- should not count when calculating the length of the returned function.

Thank you for catching this error, @CrossEye! I've updated the code and the relevant test case.